<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Vision_zedClass</class>
 <widget class="QMainWindow" name="Vision_zedClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>956</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vision_zed</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="left">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="right">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>90</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>LEFT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>40</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>right</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>1341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>70</y>
      <width>16</width>
      <height>851</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>470</y>
      <width>1341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="depth">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>520</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>540</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>打开相机</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>540</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭相机</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>720</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>录像/停止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>600</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>开始合成</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="record_timer">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>720</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>600</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>540</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>选择文件</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputPath">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>540</y>
      <width>261</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="brightness">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>610</y>
      <width>150</width>
      <height>15</height>
     </rect>
    </property>
    <property name="maximum">
     <number>8</number>
    </property>
    <property name="sliderPosition">
     <number>4</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="contrast">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>670</y>
      <width>150</width>
      <height>15</height>
     </rect>
    </property>
    <property name="maximum">
     <number>8</number>
    </property>
    <property name="sliderPosition">
     <number>4</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="saturation">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>720</y>
      <width>150</width>
      <height>15</height>
     </rect>
    </property>
    <property name="maximum">
     <number>8</number>
    </property>
    <property name="sliderPosition">
     <number>4</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="gain">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>780</y>
      <width>150</width>
      <height>15</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="sliderPosition">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>610</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>亮度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>670</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>对比度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>720</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>饱和度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>770</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>感光度</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>830</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>应用</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>830</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Vision_zed.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Vision_zedClass</receiver>
   <slot>onBtn_openCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>878</x>
     <y>609</y>
    </hint>
    <hint type="destinationlabel">
     <x>1102</x>
     <y>609</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Vision_zedClass</receiver>
   <slot>onBtn_closeCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1009</x>
     <y>609</y>
    </hint>
    <hint type="destinationlabel">
     <x>1070</x>
     <y>699</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Vision_zedClass</receiver>
   <slot>onBtn_record()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1189</x>
     <y>789</y>
    </hint>
    <hint type="destinationlabel">
     <x>1106</x>
     <y>782</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>Vision_zedClass</receiver>
   <slot>onBtn_startProcess()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1420</x>
     <y>669</y>
    </hint>
    <hint type="destinationlabel">
     <x>1125</x>
     <y>561</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>Vision_zedClass</receiver>
   <slot>onBtn_selectInputPath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1407</x>
     <y>607</y>
    </hint>
    <hint type="destinationlabel">
     <x>1192</x>
     <y>894</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>Vision_zedClass</receiver>
   <slot>onBtn_setCameraParameter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>843</x>
     <y>884</y>
    </hint>
    <hint type="destinationlabel">
     <x>914</x>
     <y>933</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>Vision_zedClass</receiver>
   <slot>onBtn_setCameraParameter_default()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>948</x>
     <y>879</y>
    </hint>
    <hint type="destinationlabel">
     <x>1219</x>
     <y>935</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBtn_openCamera()</slot>
  <slot>onBtn_closeCamera()</slot>
  <slot>onBtn_record()</slot>
  <slot>onBtn_startProcess()</slot>
  <slot>onBtn_selectInputPath()</slot>
  <slot>onBtn_setCameraParameter()</slot>
  <slot>onBtn_setCameraParameter_default()</slot>
 </slots>
</ui>
